### ISSUE TITLE
2021-11-14

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#655]
361V.1. Normalized cross entropy loss with adjustable temperature works better than alternatives
3621.compare the NT-Xent loss against other commonly used contrastive loss functions; (2021-10-27)
3632.NT-Xent loss is best; (2021-10-27)
3643.l2 normalization and temperature; without normalization and proper temperature scaling; (2021-10-27)
365V.2. Contrastive learning benefits (more) from larger batch sizes and longer training
3661.With more training steps/epochs, the gaps between different batch sizes decrease or disappear; (2021-10-27)
367VI.Comparison with State-of-the-art
3681.we use ResNet-50; (2021-10-27)
3692.Linear evaluation; best result obtained with our ResNet-50; (2021-10-27)
